src/test/controllers/users.controller.test.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import sampleUser from '../fixtures/loginedUser';
+import '../../server/models/users.model';
+
+/*
+ * Test for users.controller.
+ */
+
+let assert = require('assert');
+let should = require('should');
+let user = require('mongoose').model('user');
+
+// Test for getMentorList method.
```

Comment:
I think we can remove this line. It's in the wrong place (it should be between line 15 and 16), but line 16 already states the method name.